src/components/value-chart/ChartLoadingState.js

```diff
@@ -0,0 +1,46 @@
+import React, { useEffect, useRef } from 'react';
+import { Transition, Transitioning } from 'react-native-reanimated';
+import styled from 'styled-components/primitives';
+import ActivityIndicator from '../ActivityIndicator';
+import { Centered } from '../layout';
+import { colors, position } from '@rainbow-me/styles';
+
+const duration = 420.69;
```

Comment:
😂🤣
Love the ms duration but it's used in Transition.Sequence.
From my friend working on it, I know that transitioning will be discontinued in future releases of reanimated, and it's not recommended to use it from now on 😭

Reply:
@wojtus7 😂

yoo ,,, i’ve seen similar things mentioned in the reanimated issues/PRs whatever — 

so like,,,,, v2 currently supports Transitioning, but they plan on ditching it and replacing it with a new concept yes?

my thoughts:
1) we are still using an old version of reanimated where Transitioning is supported
2) until there’s a replacement for the Transitioning functionality, isn’t Transitioning still the best actual solution? 
3) we use Transitioning API throughout the whole app

what do you think? 
is it actually worth getting rid of Transitioning here?

Reply:
I mean it's not bad to use transitioning but as you can read for example here: 
https://github.com/software-mansion/react-native-reanimated/issues/947

There won't be any bug fixes and support in the future, and maybe will be rewritten in v2.

The main issue was that it hardly worked on android, and sometimes broke on iOS.
I'm not pushing to remove it or change it right now - more pointing out that it might be an issue in the future so we can avoid additional work just by using different approach while introducing new code 🙈